vagen/tests/test_concat_val_multi_turn.py

```
@@ -0,0 +1,283 @@
+    }
+
+
+def _make_output(rows):
+    prompts, responses, masks, rms = [], [], [], []
+    groups, trajs, turns, imgs, reis = [], [], [], [], []
+    for r in rows:
+        prompts.append(_left_pad(r["prompt"]))
+        responses.append(_right_pad(r["resp"]))
+        masks.append(_right_pad([1] * len(r["resp"])))
+        rms.append(_right_pad([1.0] * len(r["resp"]), pad=0.0))
+        groups.append(r["group"]); trajs.append(r["traj"]); turns.append(r["turn"])
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)